readMe.md

```diff
@@ -1,8 +1,11 @@
 ### Doppelkopf
 
-All core functionality can be found under src/core/...
+All core functionality can be found under src/core/...   
 The main game loop is in src/main
 
+### Test
+Test compilation is off by default, to activate it pass `-DADD_TESTS=ON` to cmake when loading the cmake project(for ex. in the gui)
+

```

Comment:
Also add how to make the tests and how to execute them.

Reply:
can you test wether `make test` run from the build directory builds and executes the tests?
Alternatively just do make and look for an `tests.exe` and run that.

Reply:
I had to rename the tests executable to tests_executable, so I don't have a naming conflict with the folder tests that is also created. But then it worked, so I will merge.